From d9b39d4e517a08350800cf253f7434cf4b865b81 Mon Sep 17 00:00:00 2001
From: Peter Korsgaard <peter@korsgaard.com>
Date: Mon, 2 Jun 2014 23:02:21 +0200
Subject: [PATCH 1/2] libinput: evdev.c: use correct fallback value for
 KEY_LIGHTS_TOGGLE

The kernel defines KEY_LIGHTS_TOGGLE as 0x21e, not 0x160 (which is KEY_OK).

Signed-off-by: Peter Korsgaard <peter@korsgaard.com>
---
 src/evdev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/evdev.c b/src/evdev.c
index ed5749e..3053764 100644
--- a/src/evdev.c
+++ b/src/evdev.c
@@ -42,7 +42,7 @@
 #define DEFAULT_AXIS_STEP_DISTANCE li_fixed_from_int(10)
 
 #ifndef KEY_LIGHTS_TOGGLE
-#define KEY_LIGHTS_TOGGLE 0x160
+#define KEY_LIGHTS_TOGGLE 0x21e
 #endif
 
 void
-- 
2.0.0.rc2

